snmp/tests/test_e2e_core_profiles/test_profile_sinetica_eagle_i.py

```diff
@@ -110,6 +110,7 @@ def test_e2e_profile_sinetica_eagle_i(dd_agent_check):
         'status': 1,
         'sys_object_id': '1.3.6.1.4.1.3711.24',
         'vendor': 'sinetica',
+        'device_type': 'other',
```

Comment:
the spreadsheet shows this as `pdu?` - just wanted to check in we're moving forward w/ this for now? maybe i missed a discussion determining the fate of this profile!